src/controller/postRouter.js

```diff
@@ -31,10 +33,12 @@ postRouter.get("/", async (req, res) => {
     }
 });
 
-postRouter.patch("/replies", authenticate, validateTextBody, async (req, res) => {
+postRouter.patch("/replies", authenticate, validateTextBody(), async (req, res) => {
```

Comment:
I would move the id to the url to follow the rest protocol. 

The route would be ":id/replies" so it signifies you are creating a reply for the post with id :id. Then you can grab the id with req.params.id.

Reply:
Looks like Jared has already fixed this over at `admin-delete-posts-comments`.